roles/task-shortcuts/files/uug_ansible_wrapper.py

```diff
@@ -28,7 +28,7 @@
 # checkbox
 # Map of course names to the Ansible tags
 COURSES = {'CS 101': 'cs101', 'CS 149': 'cs149', 'CS 159': 'cs159',
-           'CS 261': 'cs261', 'CS 354': 'cs354'}
+			'CS 261': 'cs261', 'CS 361': 'cs361',  'CS 354': 'cs354'}
```

Comment:
Can you fix the white space here? We want the first item on the second line to be aligned with the first item on the first line.

```suggestion
           'CS 261': 'cs261', 'CS 361': 'cs361',  'CS 354': 'cs354'}
```

Reply:
Sure thing, I think vim had a fit with that.